integration_tests/__tests__/__snapshots__/console.test.js.snap

```diff
@@ -43,8 +43,8 @@ exports[`console printing with --verbose 1`] = `
 `;
 
 exports[`console printing with --verbose 2`] = `
-"PASS __tests__/console.test.js
-  ✓ works just fine
+"  ✓ works just fine
+PASS __tests__/console.test.js
```

Comment:
I don't think this is the right behavior. We should print the PASS message before any test results.